inlong-sort/sort-connectors/pom.xml

```diff
@@ -39,15 +39,14 @@
         <module>kafka</module>
         <module>jdbc</module>
         <module>pulsar</module>
```

Comment:
Does filesystem need to add？

Reply:
no need.File System is in blink runtime.